internal/server/singleprocess/state/project.go

```diff
@@ -350,8 +351,9 @@ func (s *State) projectDelete(
 // projectIndexSet writes an index record for a single project.
 func (s *State) projectIndexSet(txn *memdb.Txn, id []byte, value *pb.Project) error {
 	record := &projectIndexRecord{
-		Id:   string(id),
-		Poll: false, // being explicit that we want to default poll to false
+		Id:      string(id),
+		Poll:    false, // being explicit that we want to default poll to false
+		AppPoll: true,  // application polling off by default until a deployment or release happens
```

Comment:
Should this be false?

Reply:
Yep.... This was originally set to true when I was thinking we'd have app poll settings for every app. Essentially the apps wouldn't poll until the `AppPoll{}` struct existed inside a specific app record, but app polling for a project as a whole could be switched on and off.

Anyway TL;DR we decided to make app polling a single setting across a project so yes! This should be fixed up